internal/pkg/service/stream/storage/level/local/diskwriter/network/transport/transport.go

```diff
@@ -0,0 +1,37 @@
+// Package transport provides transport layer for communication between source and disk writer nodes.
+//
+// Communication is based on KCP Reliable UDP protocol and yamux streams multiplexer.
+// TCP is an old protocol with several issues regarding latency.
+// It is not possible to make major changes to TCP, as it is a widely used standard.
+// Therefore, R-UDP protocols are used to address issues.
+package transport
+
```

Comment:
Is it clear, or do you have any questions?

Reply:
`kcp-go` is used also by the popular https://github.com/syncthing/syncthing

Reply:
In the [next PR](https://github.com/keboola/keboola-as-code/pull/1895) I slight improved code readability.

- [TCP version](https://github.com/keboola/keboola-as-code/blob/michaljurecko-PSGO-591-network-transport-2/internal/pkg/service/stream/storage/level/local/diskwriter/network/transport/transport_tcp.go)
- [KCP version](https://github.com/keboola/keboola-as-code/blob/michaljurecko-PSGO-591-network-transport-2/internal/pkg/service/stream/storage/level/local/diskwriter/network/transport/transport_kcp.go)